app/src/main/java/org/oppia/app/home/HomeFragmentPresenter.kt

```diff
@@ -64,4 +67,9 @@ class HomeFragmentPresenter @Inject constructor(
       }
     })
   }
+
+  fun openStory(v: View) {
```

Comment:
Normally I would advise that we only do this level of routing at the activity level, but in this case it's temporary. Please add a TODO that references #134 to remove this method once it's possible to navigate to StoryActivity via the TopicActivity via the home fragment.

Reply:
Done.